x/evidence/keeper/abci.go

```diff
@@ -4,33 +4,40 @@ import (
 	"context"
 	"fmt"
 
-	"cosmossdk.io/core/comet"
 	"cosmossdk.io/x/evidence/types"
 
 	"github.com/cosmos/cosmos-sdk/telemetry"
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	consensusv1 "github.com/cosmos/cosmos-sdk/x/consensus/types"
```

Comment:
this usage of types seems correct, the gogo types in x/consensus, but slashing and staking are using the API module?